charts/fluentd-elasticsearch/templates/daemonset.yaml

```diff
@@ -151,6 +151,8 @@ spec:
           value: {{ .Values.elasticsearch.reloadConnections | quote }}
         - name: OUTPUT_REQUEST_TIMEOUT
           value: {{ .Values.elasticsearch.requestTimeout | quote }}
+        - name: OUTPUT_SUPPRESS_TYPE_NAME
```

Comment:
I'd like to see a "{{- if .Values.elasticsearch.suppressTypeName }}" here, to get sure not to change stuff for people who are updateing.